docs/rules/no-unnecessary-act.md

```diff
@@ -0,0 +1,94 @@
+# Disallow wrapping Testing Library utils or empty callbacks in `act` (`testing-library/no-unnecessary-act`)
+
+> ⚠️ The `act` method is only available on the following Testing Library packages:
+>
+> - `@testing-library/react` (supported by this plugin)
+> - `@testing-library/preact` (not supported yet by this plugin)
+> - `@testing-library/svelte` (not supported yet by this plugin)
```

Comment:
Should we include ReactTestUtils.act from `react-dom/test-utils` ? It was mentioned in the original issue

Reply:
ok no, it is not part of Testing Library packages 😅 , but it should work the same according to the issue

Reply:
`react-dom/test-utils` is mentioned in the rule doc, reported by the rule, and covered in test cases.